src/__tests__/element-queries.js

```diff
@@ -129,6 +129,114 @@ test('can get elements by data-testid attribute', () => {
   expect(queryByTestId('first-name')).not.toBeInTheDOM()
 })
 
+test('queries passed strings match case-sensitive exact strings', () => {
```

Comment:
For these two test cases, I think we could improve things slightly with jest-in-case. Could you look into that please?